daemon/cluster/noderunner.go

```diff
@@ -217,7 +219,10 @@ func (n *nodeRunner) watchClusterEvents(ctx context.Context, conn *grpc.ClientCo
 		msg, err := watch.Recv()
 		if err != nil {
 			// store watch is broken
-			logrus.WithError(err).Error("failed to receive changes from store watch API")
+			errStatus, _ := status.FromError(err)
```

Comment:
`status.FromError()` can return nil; should we check the `ok` value here? https://github.com/grpc/grpc-go/blob/master/status/status.go#L131

Reply:
Done